client/src/components/HomePage/modal/JoinModal/index.jsx

```diff
@@ -11,23 +11,33 @@ import LoadingModal from '../LoadingModal';
 class Join extends Component {
   state = {};
 
-
   setPopUp = (message, title, context) => {
     context.setPopUpMessage({
-      message, title,
+      message,
+      title,
     });
     this.setState({ signingUp: false });
-  }
+  };
 
   joinCheck = (context) => {
     this.setState({ signingUp: true });
     setTimeout(() => {
       const {
-        firstName, lastName, email, password,
+        firstName, lastName, email, password, confirmPassword,
       } = context.data;
       if (firstName && lastName && email && password) {
         if (!validator.isEmail(email)) {
-          this.setPopUp('please Enter valid email adress !', 'Error !', context);
+          this.setPopUp(
+            'please Enter valid email adress !',
+            'Error !',
+            context,
+          );
+        } else if (!validator.equals(password, confirmPassword)) {
+          this.setPopUp(
+            'Passwords do not match',
+            'Error !',
```

Comment:
@HemaSAli It looks like "title" is always set to "Error!", so you could just hard-code that inside setPopUp() so that you don't need to rewrite "Error!" every time you create a new popup.
Another alternative is to give it a default value of "Error!" so that callers could still pass a unique title whenever they need.  That would require rearranging to the order of the arguments to put title last - something like:
`setPopUp = (context, message, title = 'Error!') => {`

It's not too important but might make it easier when creating new popups.